manifests/vhost.pp

```diff
@@ -539,11 +539,19 @@
     }
   }
 
+  # Check if mod_env is required and not yet loaded.
+  # create an expression to simplify the conditional check
+  $use_env_mod = $setenv and ! empty($setenv)
+  if ($use_env_mod) {
+    if ! defined(Class['apache::mod::env']) {
+      include ::apache::mod::env
+    }
+  }
   # Check if mod_setenvif is required and not yet loaded.
   # create an expression to simplify the conditional check
-  $use_setenv_mod = ($setenv and ! empty($setenv)) or ($setenvif and ! empty($setenvif)) or ($setenvifnocase and ! empty($setenvifnocase))
+  $use_setenvif_mod = ($setenvif and ! empty($setenvif)) or ($setenvifnocase and ! empty($setenvifnocase))
```

Comment:
Previously mod_setenvif was loaded when only setenv was used, which is not needed at all.

The new behavior handle the modules independently (see [mod_env](http://httpd.apache.org/docs/2.4/mod/mod_env.html) and [mod_setenvif](http://httpd.apache.org/docs/2.4/mod/mod_setenvif.html)).

So yes: behavior desired.